GUI crash when new column sent from backend

I'm looking int to this, but I wanted to record the traceback first.

```
Traceback (most recent call last):
File "/gpfs/exfel/sw/software/xfel_anaconda3/amore-mid/conda_env/lib/python3.10/runpy.py", line 196, in _run_module_as_main
return _run_code(code, main_globals, None,
File "/gpfs/exfel/sw/software/xfel_anaconda3/amore-mid/conda_env/lib/python3.10/runpy.py", line 86, in _run_code
exec(code, run_globals)
File "/gpfs/exfel/sw/software/xfel_anaconda3/amore-mid/conda_env/lib/python3.10/site-packages/damnit/cli.py", line 301, in
sys.exit(main())
File "/gpfs/exfel/sw/software/xfel_anaconda3/amore-mid/conda_env/lib/python3.10/site-packages/damnit/cli.py", line 239, in main
Extractor().update_db_vars()
File "/gpfs/exfel/sw/software/xfel_anaconda3/amore-mid/conda_env/lib/python3.10/site-packages/damnit/backend/extract_data.py", line 189, in __init__
assert error_info is None, error_info
AssertionError: ('Traceback (most recent call last):\n File "/gpfs/exfel/sw/software/xfel_anaconda3/amore-mid/conda_env/lib/python3.10/site-packages/damnit/ctxsupport/ctxrunner.py", line 708, in main\n ctx = ContextFile.from_py_file(args.context_file)\n File "/gpfs/exfel/sw/software/xfel_anaconda3/amore-mid/conda_env/lib/python3.10/site-packages/damnit/ctxsupport/ctxrunner.py", line 213, in from_py_file\n return ContextFile.from_str(code, str(path.absolute()))\n File "/gpfs/exfel/sw/software/xfel_anaconda3/amore-mid/conda_env/lib/python3.10/site-packages/damnit/ctxsupport/ctxrunner.py", line 219, in from_str\n exec(codeobj, d)\n File "/gpfs/exfel/u/usr/FXE/202405/p006736/Shared/amore/context.py", line 22, in \n from pumpprobe_splitter import array_split_pp\nModuleNotFoundError: No module named \'pumpprobe_splitter\'\n', -1, 0)
2024-10-17 11:01:56 INFO kafka.producer.kafka Closing the Kafka producer with 0 secs timeout.
2024-10-17 11:01:56 INFO kafka.producer.kafka Proceeding to force close the producer since pending requests could not be completed within timeout 0.
2024-10-17 11:01:56 INFO kafka.conn [IPv4 ('131.169.247.61', 9091)]>: Closing connection.
Traceback (most recent call last):
File "/gpfs/exfel/sw/software/xfel_anaconda3/amore-mid/conda_env/lib/python3.10/site-packages/damnit/gui/table.py", line 619, in handle_variable_set
col_ix = self.find_column(col_id)
File "/gpfs/exfel/sw/software/xfel_anaconda3/amore-mid/conda_env/lib/python3.10/site-packages/damnit/gui/table.py", line 479, in find_column
return self.column_index[name]
KeyError: 'ref_run'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File "/gpfs/exfel/sw/software/xfel_anaconda3/amore-mid/conda_env/lib/python3.10/site-packages/damnit/gui/table.py", line 475, in find_column
return self.column_titles.index(name)
ValueError: '' is not in list

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File "/gpfs/exfel/sw/software/xfel_anaconda3/amore-mid/conda_env/lib/python3.10/site-packages/damnit/gui/table.py", line 104, in item_changed
self.set_column_visibility(item.text(), state == Qt.Checked)
File "/gpfs/exfel/sw/software/xfel_anaconda3/amore-mid/conda_env/lib/python3.10/site-packages/damnit/gui/table.py", line 118, in set_column_visibility
column_index = self.damnit_model.find_column(name, by_title=True)
File "/gpfs/exfel/sw/software/xfel_anaconda3/amore-mid/conda_env/lib/python3.10/site-packages/damnit/gui/table.py", line 477, in find_column
raise KeyError(name)
KeyError: ''
/home/kluyvert/bin/launch-amore-mid: line 5: 2571577 Aborted (core dumped) amore-proto gui
```

--- a/damnit/gui/table.py
+++ b/damnit/gui/table.py
@@ -614,7 +614,7 @@ class DamnitTableModel(QtGui.QStandardItemModel):
 
     def handle_variable_set(self, var_info: dict):
         col_id = var_info['name']
-        title = var_info['title']
+        title = var_info['title'] or col_id
         try:
             col_ix = self.find_column(col_id)
         except KeyError:

